cmd/es-rollover/app/index_set.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2021 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package app
+
+import (
+	"fmt"
+	"strings"
+)
+
+const writeAliasFormat = "%s-write"
+const readAliasFormat = "%s-read"
+const rolloverIndexFormat = "%s-000001"
+
+type IndexSet struct {
```

Comment:
I would consider renaming this. One instance represents a single index and not a  set.

Reply:
Yeah, this name is not accurate, is one of thepending improvements I need to do to this PR.